kornia/augmentation/container/ops.py

```diff
@@ -185,42 +185,91 @@ class MaskSequentialOps(SequentialOpsInterface[Tensor]):
     """Apply and inverse transformations for mask tensors."""
 
     @classmethod
-    def transform(cls, input: Tensor, module: Module, param: ParamItem, extra_args: Dict[str, Any] = {}) -> Tensor:
+    def transform(
+        cls, input: Union[Tensor, List[Tensor]], module: Module, param: ParamItem, extra_args: Dict[str, Any] = {}
+    ) -> Union[Tensor, List[Tensor]]:
         """Apply a transformation with respect to the parameters.
 
         Args:
-            input: the input tensor.
+            input: the input tensor or list of tensors.
             module: any torch Module but only kornia augmentation modules will count
                 to apply transformations.
             param: the corresponding parameters to the module.
         """
         if isinstance(module, (K.GeometricAugmentationBase2D,)):
-            input = module.transform_masks(
-                input,
-                params=cls.get_instance_module_param(param),
-                flags=module.flags,
-                transform=module.transform_matrix,
-                **extra_args,
-            )
+            if isinstance(input, list):
+                tfm_input = []
```

Comment:
can you somehow factor this functionality into a single call `transform_masks_list` or similar name

Reply:
+1